### PR TITLE
File manager extension

### PR DESCRIPTION
**Pull request by vilapuigvila**
## Checklist

- [x] New Extension
- [x] New Test
- [ ] Changed more than one extension, but all changes are related
- [ ] Trivial change (doesn't require changelog)

### CODE REVIEW

**Comment by lfarah** [comment]
Can you please remove your changes in ```UIDeviceTests.swift``` and ```UIDeviceExtensions.swift``` ?

File: Sources/FileManagerExtension.swift [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+//  Created by Albert Vila on 16/11/16.
+//  Copyright © 2016 Goktug Yilmaz. All rights reserved.
+//
+
+import Foundation
+
+extension FileManager {
+    
+    /// EZSE: Returns path of documents directory
+    public class func documentsDirectoryPath() -> String {
```
Comment (Khalian) [inline Sources/FileManagerExtension.swift]: These could very well be vars instead of class funcs. 

Also we can squash commits to 1 as it does not convey any extra information.

**Comment by vilapuigvila** [comment]
Changed public classeds for vars.

**Comment by lfarah** [comment]
@Khalian is this good now?

**Comment by Khalian** [comment]
We can squash them down to one or two commits. 

Also the build failed as danger only allows changes per class (functional + unit test). Please revert the extra line in Sources/UIDeviceExtensions.swift

**Comment by vilapuigvila** [comment]
@Khalian I'm sorry, but which one is the extra line in Sources/UIDeviceExtensions.swift? I can't find it..

File: Sources/UIDeviceExtensions.swift [hunk 2]
```diff
@@ -84,7 +84,7 @@ extension UIDevice {
     public class func isPad() -> Bool {
         return UIDevice.current.userInterfaceIdiom == UIUserInterfaceIdiom.pad
     }
-
+
```
Comment (Khalian) [inline Sources/UIDeviceExtensions.swift]: Here. Revert this change.

**Comment by Khalian** [comment]
Ok you can revert Sources/UIDeviceExtensions.swift
View changes to whatever was the older state so that build passes through. 

Danger has a rule that one Pull Request should only add to one extension. The idea is that the delta across all your commits should reference that.

**Comment by lfarah** [comment]
@vilapuigvila check this out: http://stackoverflow.com/a/2734050/2938877

**Comment by vilapuigvila** [comment]
@lfarah thanks,  I'm going to take a look

**Comment by lfarah** [comment]
Hey @vilapuigvila, you have a lot of open PRs and I understand it can be overwhelming. How can I help you?

**Comment by Khalian** [comment]
Closing this since I consolidated this in a different PR : https://github.com/goktugyil/EZSwiftExtensions/pull/338

**Comment by vilapuigvila** [comment]
@lfarah you're right, please tell me which next steps I have to do it in every PR, for instance https://github.com/goktugyil/EZSwiftExtensions/pull/300
Thanks.
